csharp/README.md

```diff
@@ -19,7 +19,7 @@ Supported platforms
 The runtime library is built as a class library, supporting targets of:
 
 - .NET 4.5+ (`net45`)
-- .NET Standard 1.1 and 2.0 (`netstandard1.1` and `netstandard2.0`)
+- .NET Standard 2.0 (`netstandard2.0`)
```

Comment:
What specifically drives the requirement for netstarndard2.0 only? If really needed, an explanation for this needs to go into the PR description.

Reply:
We were using SortedList collection to store FieldDescriptor and value pair. It was not present in 1.1.
But there was no specific reason to use this collection, so changed it to use Dictionary and added .net standard 1.1 back again. So rolled back these changes.